drivers/serial/usart_gd32.c

```diff
@@ -120,9 +130,11 @@ static void usart_gd32_poll_out(const struct device *dev, unsigned char c)
 static int usart_gd32_err_check(const struct device *dev)
 {
 	const struct gd32_usart_config *const cfg = dev->config;
-	uint32_t status = USART_STAT0(cfg->reg);
+	uint32_t status;
 	int errors = 0;
 
+	status = usart_gd32_stat(cfg->reg);
+
```

Comment:
Is there any particular reason for this change? The other two variables are initialized at declaration.

Reply:
merge to one.